lib/v2/utils/EventBusEvent.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:meta/meta.dart';
```

Comment:
@gguijarro-c-chwy These entities are part of the domain layer or where would you put them?

Reply:
@RaulUrtecho Happy to put them elsewhere if there's a "correct" way to put them. Event bus is a bit like a global bloc. 

I thought about this later, but it also doesn't make sense to define the singleton inside of main.dart - should be in its own class, and initialized in main as part of initialization. I think.

Reply:
Not changing this now, waiting for @gguijarro-c-chwy review

Reply:
I was referring to the events since they are entity classes.

But now I give a review to this library, I don't think it's even necessary to put it in the dependencies.
All the repository it's a Dart class of 62 lines and without the comments around 30 all it does is use a stream controller.

Reply:
I agree - good idea to check @RaulUrtecho ... if we can get rid of the lib dependency it's good. 

So it's basically just a global stream controller for events. ;)